wordpress-mobile/WordPress-iOS
# Media: Toolbar updates for inline picker

Fixes #7703.

This PR introduces the updated Aztec format bar for the new media picker.

![mediapicker-1](https://user-images.githubusercontent.com/4780/29562787-a971eef8-8732-11e7-937b-55e476bdd839.gif)

Not implemented in this PR:

* There are no animations for the old icons disappearing and new ones appearing in the bar
* There are no extra analytics events on the new items in the bar

To test:

* Open Aztec, and attempt inserting media:
   * From the inline picker
   * From the device library (opened from the picker)
   * From the media library (opened from the picker)
   * From the camera (opened from the picker)
* Check that you can cancel out of the picker
* Check that you can cancel out of the fullscreen picker

Marking as DO NOT MERGE, as we'll need to do new Aztec and Media Picker releases to support this.

Needs review: @SergioEstevao
